examples/busola-web-components/README.md

```diff
@@ -0,0 +1,7 @@
+# Busola Web Components Example
+
+This example demonstrates the use of custom Web Components, including the Dynamic Page and Monaco Editor. It showcases how to set attributes, properties and manage content.
+
+# Set Up Your Custom Busola Extension
+
+To set up and deploy this exmaple, follow [this]('examples/../../custom-extension/README.md) documentation.
```

Comment:
```suggestion
To apply this example in your cluster execute:
```bash
kubectl kustomize . | kubectl apply -n kyma-system -f -
```
For more information follow [this]('examples/../../custom-extension/README.md) documentation.
```

Reply:
This command is provided as an alternative in the Custom Extension README. Does it mean that this command covers all previous steps (1. Enable custom extensions in your cluster, 2. Adjust the static HTML content, etc)? Or should a user execute this one first and then follow 1., 2., 3. 4. from the example?